exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/sklearn/sklearn_train_telescope.ipynb

```diff
@@ -52,13 +52,12 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    bucket = open_bucketfs_connection(sb_config)
</pre><p>I think, <code>open_bucketfs_bucket(sb_config)</code> or something like that, might be actually more precise, because my expectation would have been with this name, you get a Service object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Agree, but I suggest adding the name changing tickets for both repos, and going with existing name for now.

Reply:
yep, perfectly fine for me